spyder/app/tests/test_mainwindow.py

```diff
@@ -2459,7 +2459,7 @@ def test_preferences_shortcut_reset_regression(main_window, qtbot):
 
     Resetting shortcut resulted in error.
     """
-    dlg, index, page = preferences_dialog_helper(qtbot, main_window,
+    dlg, __, page = preferences_dialog_helper(qtbot, main_window,
                                                  'shortcuts')
```

Comment:
This line is not aligned correctly now. Please fix that.

Reply:
Fixed.